node/connection.js

```diff
@@ -305,17 +314,17 @@ TChannelConnection.prototype.onCallResponse = function onCallResponse(res) {
         }
 
         called = true;
-
-        self.ops.popOutReq(res.id, {
-            responseId: res.id,
-            code: res.code,
-            arg1: Buffer.isBuffer(res.arg1) ?
-                String(res.arg1) : 'streamed-arg1',
-            info: 'got call response for unknown id'
-        });
+        self.popOutReq(res);
```

Comment:
Perhaps we should move this “called” state onto the prototype and perform the check in popOutReq instead, so that it is idempotent.